common/utils.go

```diff
@@ -133,11 +133,20 @@ func GetWeaveNetDevs(processID int) ([]NetDev, error) {
 	})
 }
 
-// Get the weave bridge interface
-func GetBridgeNetDev(bridgeName string) ([]NetDev, error) {
-	return FindNetDevs(1, func(link netlink.Link) bool {
-		return link.Attrs().Name == bridgeName
+// Get the weave bridge interface.
+func GetBridgeNetDev(bridgeName string) (NetDev, error) {
+	var netdev NetDev
+
+	err := weavenet.WithNetNSLinkByPid(1, bridgeName, func(link netlink.Link) error {
```

Comment:
I think this code was only requesting the root namespace because it was necessary to pass something into `FindNetDevs`.  I believe all uses of `GetBridgeNetDev` are in a context where we may assume we are in the root namespace, so this isn't necessary.